redfish-core/lib/log_services.hpp

```diff
@@ -4339,6 +4345,45 @@ inline void requestRoutesSystemDumpEntryCollection(App& app)
             std::ref(app)));
 }
 
+inline void requestRoutesSystemDumpEntryDownload(App& app)
+{
+    // Hardware dump
+    BMCWEB_ROUTE(
+        app,
+        "/redfish/v1/Systems/system/LogServices/Dump/Entries/0<str>/attachment/")
+        .privileges(redfish::privileges::getLogEntry)
+        .methods(boost::beast::http::verb::get)(
+            std::bind_front(handleLogServicesDumpEntryComputerSystemDownloadGet,
+                            std::ref(app), "System"));
+
+    // Hostboot dump
+    BMCWEB_ROUTE(
+        app,
+        "/redfish/v1/Systems/system/LogServices/Dump/Entries/2<str>/attachment/")
+        .privileges(redfish::privileges::getLogEntry)
+        .methods(boost::beast::http::verb::get)(
+            std::bind_front(handleLogServicesDumpEntryComputerSystemDownloadGet,
+                            std::ref(app), "System"));
+
+    // SBE dump
+    BMCWEB_ROUTE(
+        app,
+        "/redfish/v1/Systems/system/LogServices/Dump/Entries/3<str>/attachment/")
+        .privileges(redfish::privileges::getLogEntry)
+        .methods(boost::beast::http::verb::get)(
+            std::bind_front(handleLogServicesDumpEntryComputerSystemDownloadGet,
+                            std::ref(app), "System"));
+
+    // OCMB SBE dump
+    BMCWEB_ROUTE(
+        app,
+        "/redfish/v1/Systems/system/LogServices/Dump/Entries/4<str>/attachment/")
```

Comment:
Instead of these 0-4 which have some special meaning, can we use the name here?
"/redfish/v1/Systems/system/LogServices/Dump/Entries/ocmb<str>/attachment/"

Even better though I feel is 

"/redfish/v1/Systems/system/LogServices/OCMBDump/Entries/<str>/attachment/"

Can we move to that?